spec/index.html

```diff
@@ -1511,61 +1511,70 @@ <h2>Fragment Identifiers</h2>
     same primary resource.</p>
 </section>
 
-<section id="section-generalized-rdf" class="informative">
+<section id="section-generalizations-of-rdf" class="informative">
   <h2>Generalizations of RDF Triples, Graphs, and Datasets</h2>
 
   <p>It is sometimes convenient to loosen the requirements
     on <a>RDF triples</a>.  For example, the completeness
     of the RDFS entailment rules is easier to show with
     a notion of symmetric RDF triples.</p>
 
+  <section id="section-symmetric-rdf">
+  <h3>Symmetric RDF</h3>
   <p>
     A <dfn class="export">symmetric RDF triple</dfn> allows the subject to be
     any <a>RDF term</a> that is allowed in the object position, one of
-    an <a>IRI</a>, 
+    an <a>IRI</a>,
     a <a>blank node</a>,
-    a <a>literal</a> 
-    or a <a>triple term</a>.
+    a <a>literal</a>,
+    or a <a>triple term</a>,
+    which may itself be a symmetric RDF triple.
```

Comment:
Maybe a matter of taste.
```suggestion
    or a <a>triple term</a>
    (which may itself be a symmetric RDF triple).
```